### ISSUE TITLE
PornHub: Extracting videos from GIF subcategory

### ISSUE THREAD

**Issue by potemkinhr** [yt-dlp/yt-dlp#7678]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm requesting a site-specific feature
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Croatia

### Example URLs

https://www.pornhub.com/gif/40162861

### Provide a description that is worded well enough to be understood

Is is possible to include downloading videos from the GIF portion of the site? They are effectively cut-down videos but are by default presented in video format but are not supported by default in yt-dlp. When inspecting the source of the site (see screenshot below) they are actually available in _webp_ and _mp4_ formats directly so they could be downloaded without any recoding necessary.

![ferret](https://github.com/yt-dlp/yt-dlp/assets/140313251/80b23876-bc5a-428a-b149-899e1e521d81)

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['https://www.pornhub.com/gif/38661021', '-vU']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.22621-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 6.0-full_build-www.gyan.dev (setts), ffprobe 6.0-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.18.0, brotli-1.0.9, certifi-2023.05.07, mutagen-1.46.0, sqlite3-2.6.0, websockets-11.0.3
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
Current Build Hash: 5ff3e702171a50175c34397494e2d18ce35d771c2110b1e59bd173ec2fb352aa
yt-dlp is up to date (stable@2023.07.06)
[PornHubPagedVideoList] Extracting URL: https://www.pornhub.com/gif/38661021
[download] Downloading playlist: gif/38661021
[PornHubPagedVideoList] gif/38661021: Downloading page 1
[PornHubPagedVideoList] Playlist gif/38661021: Downloading 0 items
[download] Finished downloading playlist: gif/38661021
```

**Comment by bashonly** [comment]
In case anyone else was wondering/concerned about what they just saw: that is a human hand holding a ferret.

**Comment by potemkinhr** [comment]
> In case anyone else was wondering/concerned about what they just saw: that is a human hand holding a ferret.

Although the site itself is not so family-and-work friendly at least I can provide an example that is and can draw out a smile

**Comment by Hrxn** [comment]
Are they always 720p only (even if the source video is 1080p)?

**Comment by potemkinhr** [comment]
> Are they always 720p only (even if the source video is 1080p)?

Indeed, all the GIFs are generated at a maximum resolution of 720p regardless if the source video is of a higher resolution

**Comment by mo-han** [comment]
well, gallery-dl already added this feature, so if you need a tool to download those gifs, just use that for now.
